index.js

```diff
@@ -7,12 +8,12 @@ function ghGot(path, opts) {
 		return Promise.reject(new TypeError(`Expected \`path\` to be a string, got ${typeof path}`));
 	}
 
-	const env = process.env;
+	const {env} = process;
```

Comment:
This was broken on `master` and unrelated to this PR. Please let me know if a separate fix PR is preferred for this, however I added these in separate commits for easy merge.